### ISSUE TITLE
Possible logical/computational error in createMedia.py

### ISSUE THREAD

**Issue by mcara** [spacetelescope/drizzlepac#94]
CC: @stsci-hack 

The code in a couple of `IF` blocks is never executed and so I wonder if it should be removed or whether this is a mistake and for some "units" the *desired* code is not executed... See https://github.com/spacetelescope/drizzlepac/blob/master/lib/drizzlepac/createMedian.py#L169:

```python
        if lthresh is not None:
            if proc_units.lower() == 'native':
                if native_units.lower() == "counts":
                    lthresh = lthresh * det_gain
                    if native_units.lower() == "counts/s":
                        lthresh = lthresh * img_exptime
        if hthresh is not None:
            if proc_units.lower() == 'native':
                if native_units.lower() == "counts":
                    hthresh = hthresh * det_gain
                    if native_units.lower() == "counts/s":
                        hthresh = hthresh * img_exptime
```
Code that follows `if native_units.lower() == "counts/s"` is never executed because `native_units` cannot be both `"counts"` AND `"counts/s"` at the same time. I wonder what was the intention here and what is the correct thing to do...

**Comment by mcara** [comment]
Yeah, the more I look at this code the more it seems that this is a bug. When user specifies either `combine_lthresh` or `combine_hthresh` in the `CREATE MEDIAN IMAGE` step, these limits/thresholds are not converted correctly to electrons (processing unit).

1. When `native_units` are `'COUNTS/S'` threshold does not get converted to electrons because of indentation problem in the code;
2. When `native_units` are `'ELECTRONS/S'` (e.g., WFC3) threshold does not get converted to electrons because this option is missing.

**Comment by stsci-hack** [comment]
This error would only impact WFPC2, STIS and NICMOS data, as both ACS and WFC3 have native_units of ELECTRONS or ELECTRONS/S.  However, I believe this logic should be revised to un-indent the second if-statement in each block to replicate if-elif logic.

**Comment by mcara** [comment]
@stsci-hack What about WFC3 when `native_units` are `'ELECTRONS/S'`? This code appears to skip converting  `e/s` to `e`.

**Comment by stsci-hack** [comment]
The data coming into this step should always be rate images anyway, so for those cases where the native units are already e/s, there should be no need to perform any additional transformation (unless I am not thinking clearly, which is always a possibility lately...).

**Comment by mcara** [comment]
> _"... for those cases where the native units are already e/s, there should be no need to perform any additional transformation"_

Are you sure? I think when units are `e/s` they must be converted to `e` (multiplied by exposure time)...
